react/src/utils/Uuid.js

```diff
@@ -0,0 +1,15 @@
+// -------------------------------------------------------------------------------------------------
+// UUID creation function.
+// -------------------------------------------------------------------------------------------------
+
+import uuid from 'react-uuid';
+
+function Uuid() {
+    return uuid();
+}
```

Comment:
Again not clear what the point of redefinition here is, if you're just calling a standard library function (and doing  nothing else to it) it's probably best to just use it as is.

Reply:
This one I agree is not necessary.